src/couch_replicator/test/eunit/couch_replicator_many_leaves_tests.erl

```diff
@@ -251,7 +169,33 @@ add_attachments(SourceDb, NumAtts, [{DocId, NumConflicts} | Rest]) ->
         [],
         SourceDocs
     ),
-    {ok, UpdateResults} = couch_db:update_docs(SourceDb, NewDocs, []),
+    {ok, UpdateResults} = fabric:update_docs(SourceDb, NewDocs, [?ADMIN_CTX]),
     NewRevs = [R || {ok, R} <- UpdateResults],
     ?assertEqual(length(NewDocs), length(NewRevs)),
     add_attachments(SourceDb, NumAtts, Rest).
+
+make_att(Id, Pos, Rev, Size) ->
+    AttData = crypto:strong_rand_bytes(Size),
```

Comment:
I don't have numbers, but I feel that strong_rand_bytes is slow.

Do we really need it to be random?

```
random(N) ->
   list_to_binary(lists:map(fun(_) -> rand:uniform(255) end, lists:seq(1, N))).
   
bytes(N) ->
   binary:copy(<<1>>, N).

% return correct number of bytes only if N is power of 2
power_of_two(N) ->
     binary:copy(<<1,2,3,4,5,6,7,8>>, N bsr 3).
```

While looking for benchmarks of `crypto:strong_rand_bytes/1` I came across an interesting post (require otp25) https://www.erlang.org/blog/faster-rand/#measurement-results

Reply:
I don't have strong opinion on this one. strong_rand_bytes would work fine.

Reply:
Ignore me. I see `strong_rand_bytes` in the previous version. Let's keep it then.

Reply:
It's a 100 bytes only. Sometimes we just want it be incompressible so random-like is enough. But given the small size and only one line of code it probably doesn't make too much difference.

```
4> element(1, timer:tc(fun() -> [crypto:strong_rand_bytes(100) || _ <- lists:seq(1,1000000)], ok end)) / 1000000.0.
3.021024
```

3 microseconds